plugins/@grouparoo/mongo/src/lib/disconnect.ts

```diff
@@ -1,5 +1,5 @@
 import { DisconnectPluginAppMethod } from "@grouparoo/core";
 
 export const disconnect: DisconnectPluginAppMethod = async ({ connection }) => {
-  await connection.client.close();
+  await connection.close();
```

Comment:
What does this change do?  This method should be called already whenever appOptions change... perhaps the problem was that we weren't fully disconnecting the old mongo connections with old appOptions before making a new connection?

Reply:
according to my tests when appOptions change both methods are been called normally `connect` and `disconnect`, I also made a query inside the `connect` method and it works well considering the new database. So, I think the problem might be when updating the `connection` reference across the app. So, the solution I found was to follow what is made with postgres plugin.
Also, the connection is made with the mongodb server, so you can access different databases inside this server with the same connection.

Reply:
I guess since the database is never set on the connection, it will be clear that you need to choose a database for each query (and you won't see any data until you do) 👍